drivers/espi/Kconfig.it8xxx2

```diff
@@ -21,6 +21,81 @@ config ESPI_PERIPHERAL_HOST_IO
 config ESPI_PERIPHERAL_DEBUG_PORT_80
 	default y
 
+config ESPI_PERIPHERAL_EC_HOST_CMD
+	default y
+
+choice IT8XXX2_H2RAM_HC_SIZE_CHOICE
+	prompt "choose a H2RAM space for ec host command"
+	depends on ESPI_PERIPHERAL_EC_HOST_CMD
+config IT8XXX2_H2RAM_HC_SIZE_256
+	bool "256"
+config IT8XXX2_H2RAM_HC_SIZE_16
+	bool "16"
+config IT8XXX2_H2RAM_HC_SIZE_32
+	bool "32"
+config IT8XXX2_H2RAM_HC_SIZE_64
+	bool "64"
+config IT8XXX2_H2RAM_HC_SIZE_128
+	bool "128"
+config IT8XXX2_H2RAM_HC_SIZE_512
+	bool "512"
+config IT8XXX2_H2RAM_HC_SIZE_1024
+	bool "1024"
+config IT8XXX2_H2RAM_HC_SIZE_2048
+	bool "2048"
+endchoice
+
+config ESPI_IT8XXX2_HC_H2RAM_SIZE
+	int
+	depends on ESPI_PERIPHERAL_EC_HOST_CMD
+	default 16 if IT8XXX2_H2RAM_HC_SIZE_16
+	default 32 if IT8XXX2_H2RAM_HC_SIZE_32
+	default 64 if IT8XXX2_H2RAM_HC_SIZE_64
+	default 128 if IT8XXX2_H2RAM_HC_SIZE_128
+	default 256 if IT8XXX2_H2RAM_HC_SIZE_256
+	default 512 if IT8XXX2_H2RAM_HC_SIZE_512
+	default 1024 if IT8XXX2_H2RAM_HC_SIZE_1024
+	default 2048 if IT8XXX2_H2RAM_HC_SIZE_2048
+
+config ESPI_PERIPHERAL_ACPI_SHM_REGION
+	default y
+
+choice IT8XXX2_H2RAM_ACPI_SHM_SIZE_CHOICE
+	prompt "choose a H2RAM space for ACPI shared memory region"
+	depends on ESPI_PERIPHERAL_ACPI_SHM_REGION
+config IT8XXX2_H2RAM_ACPI_SHM_SIZE_256
```

Comment:
same comment regarding `default` and an empty line here and below.

Reply:
Done.